frontend/public/components/utils/units.js

```diff
@@ -138,15 +138,58 @@ const humanize = units.humanize = (value, typeName, useRound = false) => {
 };
 
 export const humanizeMem = v => humanize(v, 'binaryBytes', true).string;
-export const humanizeCPU = v => humanize(v, 'numeric', true).string;
 export const humanizeNumber = v => humanize(v, 'numeric', true).string;
+export const humanizeCPU = v => humanize(v, 'numeric', true).string;
+export const dehumanizeCPU = v => {
+  const _dehumanizeCPU = _.spread((value, unit) => {
+    if (!unit) {
+      return {value: value * 1000, unit: 'm'};
+    }
+    return {value, unit};
```

Comment:
This is not guaranteed to return values that always have the same unit.

Reply:
for CPU isn't it 'm' or nothing?  What other units could there be?